README.md

```diff
@@ -50,4 +51,28 @@ enum Level {
 
 Level myVar = Level.MEDIUM    
 System.out.println(myVar)    // returns MEDIUM
-```
\ No newline at end of file
+```
+
+***Reserved Words***
+![list of reserved words](img.png)
+
+***Variable Types***
+
+**Primitive** - There are 8 types that all represent the actual value of the variable:
+1. Boolean
```

Comment:
boolean vs Boolean:
boolean is primitive
Bolean is an object (ex. Boolean.TRUE)

Reply:
Ah yes I wasn't thinking of the significance in capitals, will fix these!